crates/compiler/roc_target/src/lib.rs

```diff
@@ -159,56 +160,181 @@ impl From<target_lexicon::Architecture> for Architecture {
     }
 }
 
-pub const WASM_TARGET_STR: &str = "wasm32";
-pub const LINUX_X86_64_TARGET_STR: &str = "linux-x86_64";
-pub const LINUX_ARM64_TARGET_STR: &str = "linux-arm64";
-pub const MACOS_ARM64_TARGET_STR: &str = "macos-arm64";
-pub const MACOS_X86_64_TARGET_STR: &str = "macos-x86_64";
-pub const WINDOWS_X86_64_TARGET_STR: &str = "windows-x86_64";
-pub const WINDOWS_X86_32_TARGET_STR: &str = "windows-x86_32";
-pub const WIDNOWS_ARM64_TARGET_STR: &str = "windows-arm64";
```

Comment:
These constants differed from the CLI flags, which meant that `.rh` files had different names from what was specified on the command line. Now they're consistent, and `Target` is the single source of truth for these strings.